standards/countries.rst

```diff
@@ -0,0 +1,44 @@
+.. _countries:
+
+A common list of countries
+==========================
+
+Having a standardized list of country names is an important prerequisite for region
+definitions, scenario analysis and model comparison.
+
+The :class:`nomenclature` package (`read the docs`_) builds on the :class:`pycountry`
+package to provide a standardized list if country names based on the
+`ISO 3166-1 standard`_.
+
+For consistency with established conventions in the modelling community, several country
+names are shortened compared to ISO 3166-1, e.g. from "Bolivia, Plurinational State of"
+to "Bolivia". Also, "Kosovo" is added (with *alpha_3* code "KOS", following the
+`IOC <https://olympics.com/ioc/kosovo>`_), even though it is not a universally
+recognized state and not oficially included in ISO 3166-1.
+See the full list of changes on GitHub_.
+
+You can access the list of countries via the model-registration `Excel template`_.
```

Comment:
I just checked and the template currently does not reflect the most recent change of adding an ISO3 code for Kosovo. 
I opened an issue in nomenclature (https://github.com/IAMconsortium/nomenclature/issues/376)

Reply:
Good point, but not related to this PR.